src/sass/component/_banner.scss

```diff
@@ -0,0 +1,34 @@
+
+.banner {
```

Comment:
Still not convinced we need all of this css. Lets have a jam on the banner, may be we do....